mlflow/tracking/fluent.py

```diff
@@ -266,6 +267,51 @@ def end_run(status=RunStatus.to_string(RunStatus.FINISHED)):
 atexit.register(end_run)
 
 
+def mlflow_run(
+    _func=None, *, run_id=None, experiment_id=None, run_name=None, nested=False, tags=None,
```

Comment:
I am assuming start_run handles it but would decorator use + function nesting work as intended if nested=True?

That would be pretty promising.

Should we pass kwargs through in case the spec of start_run expands or not?

Reply:
It may be worth having def mlflow_run have an _mlflow_run_kwargs that is then passsed through to the start_run call.

That way, if start_run expands its spec, users can leverage the newer functionality without a new SDK release

Reply:
> I am assuming start_run handles it but would decorator use + function nesting work as intended if nested=True?
> 
> That would be pretty promising.

You are correct, that is handled by start run. So using nested decorators is just equivalent to using nested context managers.

Reply:
> It may be worth having def mlflow_run have an _mlflow_run_kwargs that is then passsed through to the start_run call.
> 
> That way, if start_run expands its spec, users can leverage the newer functionality without a new SDK release

Here, are you suggesting replacing the keyword arguments with mlflow_run_kwargs or adding it additionally? The reason that I went for the keyword argument option was for ease of code completion and inline docs usage, but this does come with the drawback that the decorator signature would have to be updated to match `start_run`. I am open to debate on this.